warehouse/templates/includes/banner-messages.html

```diff
@@ -0,0 +1,25 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+
+{% for banner in banners %}
+<div class="notification-bar  notification-bar--fundraiser">
+  <span class="notification-bar__icon">
+    <i class="fa fa-comment-alt" aria-hidden="true"></i>
+  </span>
+  <span class="notification-bar__message">
+    {{ banner.text }}&nbsp;
+    <a href="{{ banner.link_url }}" class="button button--dark button--small button--primary" target="_blank" rel="noopener">Open</a>
```

Comment:
If the "Open" wording is static, should it be translated ? (I'm not sure but it can be discussed)

Reply:
Actually, this was already replaced by `link_label` field in aa40500d27f6111e5e92eb9250de391151e6fbf8